packages/babel-preset-default/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 4.0.0 (Unreleased)
+
+## Breaking Change
+
+- Removed `babel-core` dependency acting as Babel 7 bridge ([#13922](https://github.com/WordPress/gutenberg/pull/13922).
```

Comment:
Can you explain what meaningful impact this has to call it a breaking change? Maybe a note in the changelog explaining what a consumer will need to do to adapt?

Reply:
This is a really great question. I know that it was necessary for Jest to work with Babel 7 due to internal code requirements. We had a similar issue in our test codebase where we didn't update `babel-core` to `@babel/core`. So I think the message should be:
Make sure you update all references from `babel-core` to `@babel/core`.
Would it be enough?

Reply:
>So I think the message should be:
>
>>Make sure you update all references from `babel-core` to `@babel/core`.
>
>Would it be enough?

Yeah, that sounds fine.